docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -6,7 +6,12 @@ components: DateTimePickerTabs
 
 # Custom components
 
-<p class="description">The date picker lets you customize sub-components.</p>
+<p class="description">The date picker lets you customize sub-components.
+
+:::info
+The components which can be customized are listed under `slots` section in DatePicker API Reference.
+:::
+</p>
```

Comment:
What do you think about the following suggestion. It's adding links and increasing the description specificity.
Also fixing the `p` element ending location.

```suggestion
<p class="description">The date picker lets you customize sub-components.</p>

:::info
The components that can be customized are listed under `slots` section in Date and Time Pickers [API Reference](/x/api/date-pickers/).
For example, available Date Picker slots can be found [here](/x/api/date-pickers/date-picker/#slots).
:::
```

Reply:
sure @LukasTy ill update the PR, Thanks you for the suggestion